content/en/agent/logs/proxy.md

```diff
@@ -60,17 +69,219 @@ The parameter above can also be set with the following environment variable:
+        * `sudo apt-get install ca-certificates` (Debian, Ubuntu)
+        * `yum install ca-certificates` (CentOS, Redhat)
+The file might be located at `/etc/ssl/certs/ca-bundle.crt` for CentOS, Redhat.
+
+Once the HAProxy configuration is in place, you can reload it or restart HAProxy. **It is recommended to have a `cron` job that reloads HAProxy every 10 minutes** (usually doing something like `service haproxy reload`) to force a refresh of HAProxy's DNS cache, in case `app.datadoghq.com` fails over to another IP.
+
+{{% /tab %}}
+{{% tab "Datadog EU site" %}}
+
+```conf
+# Basic configuration
+global
+    log 127.0.0.1 local0
+    maxconn 4096
+    stats socket /tmp/haproxy
 
-When the Agent is [configured to send logs through HTTPS][1], use the same [set of proxy settings][2] as the other data types in order to send logs through a web proxy.
+# Some sane defaults
+defaults
+    log     global
+    option  dontlognull
+    retries 3
+    option  redispatch
+    timeout client 5s
+    timeout server 5s
+    timeout connect 5s
+
+# This declares a view into HAProxy statistics, on port 3833
+# You do not need credentials to view this page and you can
+# turn it off once you are done with setup.
+listen stats
+    bind *:3833
+    mode http
+    stats enable
+    stats uri /
+    
+# This declares the endpoint where your Agents connects for
+# sending Logs (e.g the value of "logs.config.logs_dd_url")
+frontend logs_frontend
+    bind *:10514
+    mode tcp
+    default_backend datadog-logs
+    
+# This is the Datadog server. In effect any TCP request coming
+# to the forwarder frontends defined above are proxied to
+# Datadog's public endpoints.
+
+backend datadog-logs
+    balance roundrobin
+    mode tcp
+    option tcplog
+    server datadog agent-intake.logs.datadoghq.eu:443 ssl verify required ca-file /etc/ssl/certs/ca-bundle.crt check port 443
+```
+
+**Note**: Download the certificate with the following command:
+
+* `sudo apt-get install ca-certificates` (Debian, Ubuntu)
+* `yum install ca-certificates` (CentOS, Redhat)
+
+The file might be located at `/etc/ssl/certs/ca-bundle.crt` for CentOS, Redhat.
+
+Once the HAProxy configuration is in place, you can reload it or restart HAProxy. **It is recommended to have a `cron` job that reloads HAProxy every 10 minutes** (usually doing something like `service haproxy reload`) to force a refresh of HAProxy's DNS cache, in case `app.datadoghq.eu` fails over to another IP.
 
-[1]: /agent/logs/?tab=tailexistingfiles#send-logs-over-https
-[2]: /agent/proxy
 {{% /tab %}}
 {{< /tabs >}}
 
+#### Agent configuration
+
+Then edit the `datadog.yaml` Agent configuration file and set `logs_no_ssl` to `true`. This is needed as Haproxy does not simply forward the traffic and is not the Datadog backend so cannot use the same certificate.
+
+**Note**: `logs_no_ssl` might set to true because Haproxy is configured to encrypt the data. Do not set this parameter to `true` otherwise.
```

Comment:
```suggestion
**Note**: `logs_no_ssl` might set to true because HAProxy is configured to encrypt the data. Do not set this parameter to `true` otherwise.
```